.github/pull_request_template.md

```diff
@@ -0,0 +1,39 @@
+## What changes are proposed in this pull request?
+ 
+(Please fill in changes proposed in this fix)
+ 
+## How is this patch tested?
+ 
+(Details)
+ 
+## Release Notes
+ 
+### Is this a user-facing change? If so, please give a 1-2 sentence description for MLflow users.
+ 
+(Details in 1-2 sentences. You can just refer to another PR with a description if this PR is part of a larger change.)
+ 
+### What component(s) does this PR touch?
+ 
+- [ ] UI
```

Comment:
UI/UX (no UI code change but some change in user experience)

Reply:
does that include APIs?